ppl/src/main/java/org/opensearch/sql/ppl/parser/AstBuilder.java

```diff
@@ -106,6 +110,21 @@ public UnresolvedPlan visitSearchFilterFrom(SearchFilterFromContext ctx) {
         visit(ctx.fromClause()));
   }
 
+  /**
+   * Describe command.
+   */
+  @Override
+  public UnresolvedPlan visitDescribeCommand(DescribeCommandContext ctx) {
+    // Gets the mapping table (_ODFE_SYS_TABLE_MAPPINGS) of the indices by name
+    final String tableName = ctx.tableSource()
+        .stream()
+        .map(this::internalVisitExpression)
+        .map(UnresolvedExpression::toString)
+        .collect(Collectors.joining(","));
+    final Relation table = new Relation(qualifiedName(mappingTable(tableName)));
+    return new Project(Collections.singletonList(AllFields.of())).attach(table);
```

Comment:
Why add Project operator?. https://github.com/opensearch-project/sql/blob/main/ppl/src/main/java/org/opensearch/sql/ppl/utils/UnresolvedPlanHelper.java#L24 already done this.

Reply:
removed Project